website/README.md

```diff
@@ -1,14 +1,12 @@
 # The Vector website and documentation
 
-[![Netlify Status](https://api.netlify.com/api/v1/badges/abeaffe6-d38a-4f03-8b6c-c6909e94918e/deploy-status)](https://app.netlify.com/sites/vector-project/deploys)
-
-This directory houses all of the assets used to build Vector's website and documentation, available at [vector.dev][vector].
+This directory houses all the assets used to build Vector's website and documentation, available at [vector.dev][vector].
 
 ## Prerequisites
 
 In order to run the site [locally](#run-the-site-locally), you need to have these installed:
 
-* The [Hugo] static site generator. Make sure to install the extended version (with [Sass] and [ESBuild] support), specifically the version specified in [`netlify.toml`](../netlify.toml).
+* The [Hugo] static site generator. You can install with `brew install hugo`.
```

Comment:
I might link to. https://gohugo.io/installation/ since instructions differ depending on environment.